docs/presto-admin-commands.rst

```diff
@@ -392,9 +392,18 @@ server install
 **************
 ::
 
-    presto-admin server install <local_path> [--nodeps]
-
-This command copies the presto-server rpm from ``local_path`` to all the nodes in the cluster, installs it, deploys the general presto configuration along with tpch connector configuration. The ``local_path`` should be accessible by ``presto-admin``.
+    presto-admin server install <rpm_specifier> [--rpm-source] [--nodeps]
+
+This command takes in a parameter ``rpm_specifier``, which must be a string describing the presto-server rpm to be used.
+The string can be 'latest'. This finds a local copy or downloads of the latest version of the presto rpm.
```

Comment:
Suggest: 'The parameter can be one of the following forms, listed in order of decreasing precedence:'
and omit 'The string can be' from each form.